TRANSCRIPTION.md

```diff
@@ -380,8 +380,10 @@
 ## Words with invalid segments (up to 100 only)
```

Comment:
@LanguageStructure, you may want to fix these cases in EDICTOR, snake ends in a space in both cases, which is of course not allowed.